config/purposes/scrna_core_library_prep.wip.yml

```diff
@@ -0,0 +1,103 @@
+  :file_links: []
+# NB. QC concentrations need to be uploaded via QuantHub for the Cherrypick plate in order for the binning
+# calculations in the Dil plate step to work.
+LRC GEM-X 5p Cherrypick:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate: false
+  :cherrypickable_target: true
+  :creator_class: LabwareCreators::TenStamp
+  :state_changer_class: StateChangers::AutomaticPlateStateChanger
+  :work_completion_request_type: 'limber_scrna_core_chromium_gem_x_5p_aggregation'
+  :file_links:
+    - name: 'Download Hamilton Aggregate Cherrypick CSV'
+      id: 'hamilton_aggregate_cherrypick'
+    - name: Download Concentration (ng/ul) CSV
+      id: concentrations_ngul
+# NB. For the following binning step the min and max amounts for the bins are the total amount of cDNA
+# in the well i.e. QC concentration * (source_volume + diluent_volume)
+# Here we are using a fixed source volume and there is no dilution (diluent_volume = 0)
+LRC GEM-X 5p GE Dil:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate: false
+  :presenter_class: Presenters::ConcentrationBinnedPlatePresenter
+  :creator_class: LabwareCreators::ConcentrationBinnedPlate
+  :dilutions:
+    :source_volume: 10
+    :diluent_volume: 0
+    :bins:
+      - max: 25.0
+        pcr_cycles: 15
+        colour: 1
+      - min: 25.0
+        max: 150.0
+        pcr_cycles: 13
+        colour: 2
+      - min: 150.0
+        max: 500.0
+        pcr_cycles: 11
+        colour: 3
+      - min: 500.0
+        max: 1000.0
+        pcr_cycles: 9
+        colour: 4
+      - min: 1000.0
+        max: 1500.0
+        pcr_cycles: 7
+        colour: 5
+      - min: 1500.0
+        pcr_cycles: 5
+        colour: 12
+  :file_links:
+    - name: 'Download Hamilton LRC GEM-X 5p Cherrypick to LRC GEM-X 5p GE Dil CSV'
+      id: hamilton_lrc_gem_x_5p_cherrypick_to_lrc_gem_x_5p_ge_dil
+    - name: Download Concentration (ng/ul) CSV
+      id: concentrations_ngul
+LRC GEM-X 5p GE Frag 2XP:
+  :asset_type: plate
+  :file_links:
+    - name: 'Download Hamilton Dilution to Frag 2XP CSV'
+      id: 'hamilton_gex_dil_to_gex_frag_2xp'
+LRC GEM-X 5p GE LigXP:
+  :asset_type: plate
+  :presenter_class: Presenters::PcrPresenter
+  :creator_class: LabwareCreators::CustomTaggedPlate
+  :tags_per_well: 1
+  :file_links:
+    - name: 'Download Hamilton Frag 2XP to LigXP CSV'
+      id: 'hamilton_gex_frag_2xp_to_gex_ligxp'
+LRC GEM-X 5p GE PCR 2XP:
+  :asset_type: plate
+  :presenter_class: Presenters::PcrPresenter
+  :default_printer_type: :plate_b
+  :label_template: plate_xp
+  :file_links:
+    - name: Download Concentration (ng/ul) CSV
+      id: concentrations_ngul
+LRC GEM-X 5p GE GLibPS:
+  :asset_type: tube
+  :target: StockMultiplexedLibraryTube
+  :type: IlluminaHtp::InitialStockTubePurpose
+  :creator_class: LabwareCreators::PooledTubesBySubmission
+  :presenter_class: Presenters::SimpleTubePresenter
+  :default_printer_type: :tube
+LRC GEM-X 5p GE Pool Norm:
```

Comment:
That name is almost certainly too long to fit on a tube label, but lets check all those labels at UAT